### PR TITLE
Disable inconsitent remoted tests

### PR DESCRIPTION
**Pull request by damarisg**
|Related issue|
|---|
|#1530|

## Description

This PR disables some test that fails by https://github.com/wazuh/wazuh-qa/issues/1698 

| Test Executions | Date  | By  | Status | 
|--|--|--|--|
|[Disabled.log](https://github.com/wazuh/wazuh-qa/files/6957010/Disabled.log)| 09-08-21| Seyla| 🟡 
|[Disabled2.log](https://github.com/wazuh/wazuh-qa/files/6957011/Disabled2.log)|09-08-21| Seyla| 🟡
|[Disabled3.log](https://github.com/wazuh/wazuh-qa/files/6957108/Disabled3.log)| 09-08-21| Seyla|  🟡

## Tests

- [x] Proven that tests **pass** when they have to pass.
- [x] Proven that tests **fail** when they have to fail.

### CODE REVIEW

**Comment by DProvinciani** [comment]
I validated the expected results of the tests with the branch `1530-disable-inconsitent-tests`.

| Test Executions | Date  | By  | Status | 
|--|--|--|--|
|[test_remoted-full-r1.log](https://github.com/wazuh/wazuh-qa/files/6958227/test_remoted-full-r1.log)|09-08-21| Diego| 🟡 
|[test_remoted-full-r2.log](https://github.com/wazuh/wazuh-qa/files/6958230/test_remoted-full-r2.log)|09-08-21| Diego| 🟡
|[test_remoted-full-r3.log](https://github.com/wazuh/wazuh-qa/files/6958231/test_remoted-full-r3.log)|09-08-21| Diego|  🟡

File: tests/integration/test_remoted/test_active_response/test_active_response_send_ar.py [hunk 1]
```diff
@@ -43,13 +43,16 @@
...
 
+
 # fixtures
 @pytest.fixture(scope='module', params=configurations, ids=configuration_ids)
 def get_configuration(request):
     """Get configurations from the module."""
     return request.param
 
 
+@pytest.mark.skip(reason="It require review and rework of agent simulator, sometimes no work successfully when "
```
Comment (DProvinciani) [inline tests/integration/test_remoted/test_active_response/test_active_response_send_ar.py:54]: Just a reword of the comment...

```
It requires review and a rework for the agent simulator. Sometimes it doesn't work properly when it sends keepalives messages causing the agent to never being in active status.
```

File: tests/integration/test_remoted/test_agent_communication/test_multi_agent_status.py [hunk 2]
```diff
@@ -115,6 +115,8 @@ def get_configuration(request):
     return request.param
 
 
+@pytest.mark.skip(reason="It require review and rework of agent simulator, sometimes no work successfully when "
```
Comment (DProvinciani) [inline tests/integration/test_remoted/test_agent_communication/test_multi_agent_status.py:118]: DITTO

**Review by DProvinciani — CHANGES_REQUESTED** [review]
Please, apply changes according to the comments.

**Comment by pereyra-m** [comment]
These are results of the **remoted** tests with the `1530-disable-inconsitent-tests` branch

| Test Executions | Date  | By  | Status | 
|--|--|--|--|
|[test_full_remoted_after_disabled_1.log](https://github.com/wazuh/wazuh-qa/files/6958501/test_full_remoted_after_disabled_1.log)| 2021-08-09| Matias | :yellow_circle:
|[test_full_remoted_after_disabled_2.log](https://github.com/wazuh/wazuh-qa/files/6958502/test_full_remoted_after_disabled_2.log)| 2021-08-09| Matias | :yellow_circle:
|[test_full_remoted_after_disabled_3.log](https://github.com/wazuh/wazuh-qa/files/6958503/test_full_remoted_after_disabled_3.log)| 2021-08-09| Matias | :yellow_circle:

**Review by DProvinciani — APPROVED** [review]
Looks good!

**Pull request merged**